spec/unit/webrtc/call.spec.ts

```diff
@@ -1191,4 +1193,89 @@ describe('Call', function() {
             id: "usermedia_video_track",
         }));
     });
+
+    describe("call transfers", () => {
+        const ALICE_USER_ID = "@alice:foo";
+        const ALICE_DISPLAY_NAME = "Alice";
+        const ALICE_AVATAR_URL = "avatar.alice.foo";
+
+        const BOB_USER_ID = "@bob:foo";
+        const BOB_DISPLAY_NAME = "Bob";
+        const BOB_AVATAR_URL = "avatar.bob.foo";
+
+        beforeEach(() => {
+            mocked(client.client.getProfileInfo).mockImplementation(async (userId) => {
+                if (userId === ALICE_USER_ID) {
+                    return {
+                        displayname: ALICE_DISPLAY_NAME,
+                        avatar_url: ALICE_AVATAR_URL,
+                    };
+                } else if (userId === BOB_USER_ID) {
+                    return {
+                        displayname: BOB_DISPLAY_NAME,
+                        avatar_url: BOB_AVATAR_URL,
+                    };
+                } else {
+                    return {};
+                }
+            });
+        });
+
+        it("transfers call to another call", async () => {
+            const newCall = new MatrixCall({
+                client: client.client,
+                roomId: FAKE_ROOM_ID,
+            });
+            // call checks one of these is wired up
+            newCall.on(CallEvent.Error, () => { });
+
+            // @ts-ignore Mock
+            jest.spyOn(call, "terminate");
+            // @ts-ignore Mock
+            jest.spyOn(newCall, "terminate");
```

Comment:
Any way to test this without peeking at private methods?

Reply:
Good point!